src/main/java/com/gitee/jenkins/util/ProjectIdUtil.java

```diff
@@ -23,6 +23,11 @@ public static String retrieveProjectId(GiteeClient client, String remoteUrl) thr
             String projectId;
             if (baseUri != null && remoteUrl.startsWith(baseUri)) {
                 projectId = new URIish(remoteUrl.substring(baseUri.length())).getPath();
+                if (projectId.contains(":")) {
+                    projectId = projectId.substring(projectId.indexOf(":"));
```

Comment:
When I ran this from the Groovy script console inside Jenkins, it returned a string that starts with ':'.  Is that expected?  I assumed you want the resulting string to not include the ':' character
```suggestion
                    projectId = projectId.substring(projectId.indexOf(":") + 1);
```

I was also surprised at the removal of the preceding characters, since that seems like it would not identify the project ID correctly.

However, I don't understand Gitee well enough to be sure of either of those things and I can't read Chinese at all.  That means you are welcome to ignore my review, since it is coming from my guesses, not from anything that I've investigated.